tests/test_keras_kfoldevaluation_transform.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import unittest
+import numpy as np
+import pandas as pd
+
+from hidi.matrix import KerasKfoldTransform
+
+if sys.version_info < (3, 0):
+    from mock import Mock
+else:
+    from unittest.mock import Mock
+
+
+class TestKerasKfoldTransform(unittest.TestCase):
+    def setUp(self):
+        self.items = range(10000)
+        self.M = pd.DataFrame(np.random.rand(10000, 6), index=self.items)
+        self.M.index.name = 'items'
+        self.validation = pd.DataFrame(np.random.rand(10000, 2),
+                                       index=self.items)
+        self.validation.index.name = 'items'
+
+    def test_baselinemodel_fit_is_called_ten_times(self):
+        t = KerasKfoldTransform(Mock(), self.validation)
+        model, _ = t.transform(self.M)
+        self.assertEqual(10, model.fit.call_count)
```

Comment:
Nah. Just add the test:

```python
def test_keras_model_is_returned(self):
    mock = Mock()
    t = KerasKfoldTransform(Mock(), self.validation)
    model, _ = t.transform(self.M)
    self.assertEqual(mock, model)
```

Generally, I try to only have one assert per test, but sometimes I'm lazy.